test/e2e/serial/workload_placement_test.go

```diff
@@ -310,7 +193,7 @@ var _ = Describe("[serial][disruptive][scheduler] workload placement", func() {
 			Expect(schedOK).To(BeTrue(), "pod %s/%s not scheduled with expected scheduler %s", updatedPod.Namespace, updatedPod.Name, schedulerName)
 		})
 
-		It("[test_id:47583][tier1] a deployment with a guaranteed pod with one container should be scheduled into one NUMA zone", func() {
+		It("[test_id: 47583][tier1] a deployment with a guaranteed pod with one container should be scheduled into one NUMA zone", func() {
```

Comment:
no spaces between `test_id:` and the number, please

Reply:
right, my mistake didn't pay much attention to previous fixing PRs. fixed. thanks.